docs/tutorials/02-local-simple-blueprint.md

```diff
@@ -85,13 +85,31 @@ version: v0.1.0
 relation: external
 input:
   type: "dir" # type dir automatically creates a tar from the given dir.
-  path: /tmp/chart # path to the downloaded helm chart
+  path: /tmp/chart # parent path to the downloaded helm chart
   compress: true # compresses the tar/blob using gzip.
   # Media type automatically defaults to "application/gzip" for compressed=true
   # mediaType: "application/gzip"
 ...
 ```
 
+:warning: In the above example, the path to the helm chart points to the parent directory of `/tmp/chart/nginx-ingress`. 
+If the fully qualified path to the helm chart should be used, the option `preserveDir` has to be set to `true`.
```

Comment:
maybe we should also add an explanation why this is needed -> helm structure enforces a parent directory

Reply:
👍 What about now?